main.js

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Very strange pattern you're using here. If it can be called so. 
https://toddmotto.com/mastering-the-module-pattern/
Please read this or any other resources and refactor it.